### PR TITLE
Add localization utils

### PR DESCRIPTION
**Pull request by bgerstle**
Added two scripts for checking common localization mistakes:

- Using templates outside **qqq.lproj**
- Reversing dollar sign substitution variables (i.e. `1$` instead of `$1`)

Current output of `lint-localizations.sh`, to be fixed separately:

> Wikipedia/Localizations/cs.lproj/Localizable.strings:"nearby-distance-label-meters" = "$1 {{PLURAL:$1|metr|metry|metrů}}";
Wikipedia/Localizations/cs.lproj/Localizable.strings:"nearby-distance-label-feet" = "$1 {{PLURAL:$1|stopa|stopy|stop}}";
Wikipedia/Localizations/cs.lproj/Localizable.strings:"nearby-distance-label-miles" = "$1 {{PLURAL:$1|míle|míle|mil}}";
Wikipedia/Localizations/ksh.lproj/Localizable.strings:"article-languages-downloading" = "Ben dä Sigg iehr Schprohche aam ahde&nbsp;{{int:ellipsis}}";
Wikipedia/Localizations/ksh.lproj/Localizable.strings:"account-creation-saving" = "Ben aam Afseschere{{int:ellipsis}}";
Wikipedia/Localizations/ksh.lproj/Localizable.strings:"lastmodified-by-anon" = "{{ucfirst:$1}} vun enem nahmelohse Metmaacher verändert.";
Wikipedia/Localizations/pl.lproj/Localizable.strings:"edit-summary-title" = "Jak {{GENDER:|poprawiłeś|poprawiłaś}} stronę?";
Found files containing templates!

### CODE REVIEW

File: Wikipedia/Localizations/azb.lproj/Localizable.strings [hunk 1]
```diff
@@ -46,7 +46,7 @@
 "search-recent-clear-cancel" = "وازگئچ";
 "search-recent-clear-delete-all" = "هامیسین سیل";
 "main-menu-title" = "آرتیق";
-"main-menu-language-title" = "ویکی‌پئدیادا 1$ آختار";
+"main-menu-language-title" = "ویکی‌پئدیادا $1 آختار";
```
Comment (bgerstle) [inline Wikipedia/Localizations/azb.lproj/Localizable.strings]: note, this looks wrong in GitHub web UI and Xcode. you need to `cat` the file or view in Terminal editor to see the actual string data.

**Comment by coreyfloyd** [comment]
Do we run this manually or does it run as part of the language pr or as part of the update qqq or something else?

**Comment by bgerstle** [comment]
Basically need to be run manually, since the exec args are slightly different on whatever betawiki is running and OS X. I'd like to make the linting script part of CI, but it will currently break due to template usage, which I don't want to address w/o figuring out how to flag the offending strings in TWN

**Comment by bgerstle** [comment]
@coreyfloyd see https://phabricator.wikimedia.org/T123300

**Comment by coreyfloyd** [comment]
gotcha :+1:

**Pull request merged**